state-of-k8s-cost-optimization/README.md

```diff
@@ -0,0 +1,12 @@
+# kube-requests-checker
```

Comment:
In the README, could we also point to the live c.g.c doc that this sample is used in?

Reply:
This won't be in CGC doc, but rather linked in a CGC blog going out next week. Once the blog is live can I open a separate PR and update the README?